devenv/local/docker-compose/docker-compose.yml

```diff
@@ -1,41 +1,54 @@
+  - &BITCOIN_PEER_PORT 18444
+  - &BITCOIN_RPC_PORT 18443
+  - &BITCOIN_RPC_USER devnet
+  - &BITCOIN_RPC_PASS devnet
+  - &MINE_INTERVAL ${MINE_INTERVAL:-1s}
+  - &MINE_INTERVAL_EPOCH25 ${MINE_INTERVAL_EPOCH25:-10s} # 10 second bitcoin block times in epoch 2.5
+  - &MINE_INTERVAL_EPOCH3 ${MINE_INTERVAL_EPOCH3:-30s} # 10 minute bitcoin block times in epoch 3
+  - &NAKAMOTO_BLOCK_INTERVAL 2 # seconds to wait between issuing stx-transfer transactions (which triggers Nakamoto block production)
+  - &STACKS_20_HEIGHT ${STACKS_20_HEIGHT:-0}
+  - &STACKS_2_05_HEIGHT ${STACKS_2_05_HEIGHT:-102}
+  - &STACKS_21_HEIGHT ${STACKS_21_HEIGHT:-103}
+  - &STACKS_POX2_HEIGHT ${STACKS_POX2_HEIGHT:-104} # 104 is is stacks_block=1, 106 is stacks_block=3
+  - &STACKS_22_HEIGHT ${STACKS_22_HEIGHT:-105}
+  - &STACKS_23_HEIGHT ${STACKS_23_HEIGHT:-106}
+  - &STACKS_24_HEIGHT ${STACKS_24_HEIGHT:-107}
+  - &STACKS_25_HEIGHT ${STACKS_25_HEIGHT:-108}
+  - &STACKS_30_HEIGHT ${STACKS_30_HEIGHT:-131}
+  - &STACKING_CYCLES ${STACKING_CYCLES:-1} # number of cycles to stack-stx or stack-extend for
+  - &POX_PREPARE_LENGTH ${POX_PREPARE_LENGTH:-5}
+  - &POX_REWARD_LENGTH ${POX_REWARD_LENGTH:-20}
+  - &REWARD_RECIPIENT ${REWARD_RECIPIENT:-STQM73RQC4EX0A07KWG1J5ECZJYBZS4SJ4ERC6WN} # priv: 6ad9cadb42d4edbfbe0c5bfb3b8a4125ddced021c4174f829b714ccbf527f02001
+  - &EXIT_FROM_MONITOR 1 # set to "1" to automatically shut down via monitor.ts
+
 services:
   bitcoin:
-    image: bitcoin:latest
-    container_name: bitcoin
-    stop_grace_period: 5s
     build:
-      context: ./bitcoin/docker
-      args:
-        VERSION: "25.0"
-        BTC_NETWORK: "regtest"
-        BTC_PRINTTOCONSOLE: 1
-        BTC_DISABLEWALLET: 0
-        BTC_TXINDEX: 1
-        BTC_RPCBIND: "0.0.0.0"
-        BTC_RPCALLOWIP: "0.0.0.0"
-        BTC_RPCPASSWORD: "devnet"
-        BTC_RPCUSER: "devnet"
-        BTC_RPC_PORT: 18443
-        BTC_P2P_PORT: 18444
-        BTC_LOG_DEBUG: 0
-        BTC_REST_ENABLE: 0
+      context: .
+      dockerfile: Dockerfile.btc
     ports:
-      - 18444:18444
-      - 18443:18443
-      - 18433:18433
-    environment:
-      BTC_NETWORK: regtest
-      BTC_PRINTTOCONSOLE: 1
-      BTC_DISABLEWALLET: 0
-      BTC_TXINDEX: 1
-      BTC_RPCBIND: "0.0.0.0"
-      BTC_RPCALLOWIP: "0.0.0.0/0"
-      BTC_RPCPASSWORD: devnet
-      BTC_RPCUSER: devnet
-      BTC_RPC_PORT: 18443
-      BTC_P2P_PORT: 18444
-      BTC_LOG_DEBUG: 0
-      BTC_REST_ENABLE: 0
+      - "18443:18443"
+      - "18444:18444"
+    volumes:
+      - ./bitcoin.conf:/root/.bitcoin/bitcoin.conf
+      - ./init-data:/init-data
+      - chainstate:/chainstate
+    environment:
+      DATA_DIR: /chainstate/bitcoin-data
+    entrypoint:
+      - /bin/bash
+      - -c
+      - |
+        set -e
+        mkdir -p $${DATA_DIR}
+        rm -rf $${DATA_DIR}/*
```

Comment:
Does it matter if this is cleared each run but the sqlite database is not?

Reply:
Can you elaborate - do you mean Stacks Core's DBs? Because those have the same command, too. I'm probably missing something

Reply:
> Can you elaborate - do you mean Stacks Core's DBs?

I was referring to the signers' databases, which didn't appear to be wiped. 

Another non-critical question cuz I'm curious, why have the volume if we're going to wipe it on boot anyway? Why not just use `/tmp` as the data directory?